src/components/products-page/ProductsPage.jsx

```diff
@@ -0,0 +1,22 @@
+import productsData from '../../assets/data.json'
+import { useState } from 'react';
+import ProductTable from './product-table/ProductTable';
+import SearchBar from  './search-bar/SearchBar'
+
+function ProductsPage() {
+    const [products, setProducts] = useState(productsData)
+
+    const handleSearch = (search) => setProducts(productsData.filter((product) => product.name.toLocaleLowerCase().includes(search.toLocaleLowerCase())))
```

Comment:
```suggestion
    const handleSearch = (search) => setProducts(productsData.filter((product) => product.name.toLocaleLowerCase().includes(search.toLocaleLowerCase())))
```

Esta perfect! pero cuando el codigo empieza a ser tan largo te recomiendo que le pongas algunos saltos de linea, sobre todo porque algunos lintern no suelen pasarte esto vale?